src/jit/morph.cpp

```diff
@@ -7168,6 +7168,7 @@ bool Compiler::fgCanFastTailCall(GenTreeCall* callee)
     }
 
     const unsigned maxRegArgs = MAX_REG_ARG;
+    hasTwoSlotSizedStruct     = hasTwoSlotSizedStruct || info.compHasMultiSlotArgs;
```

Comment:
This should be written as:

`hasTwoSlotSizedStruct     |= info.compHasMultiSlotArgs;`

Reply:
I'm not sure it needs to be rewritten, but if it does, it should be:
```
hasTwoSlotSizedStruct ||= info.compHasMultiSlotArgs;
```
since `hasTwoSlotSizedStruct` is a `bool`.

Reply:
Personally I prefer it as it is, but i'm good either way.

Reply:
There is no such operator as ||=
These two || && are short-circuiting forms of the standard operators | and &
I only use thme when there is some value is having a short-circuit evaluation.
With two bool values there is no reason not to simply use |=

I don't feel that this has to be changed, so if you want to check this in with a short-circuit form,
 I guess that is OK.